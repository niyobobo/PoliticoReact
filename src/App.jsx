import React, { Component } from 'react';

class App extends Component {
  state = {}
  render() {
    return (
      <div>
        <h1>My React app y</h1>
      </div>
    );
  }
}

export default App;