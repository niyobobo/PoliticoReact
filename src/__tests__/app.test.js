import React from 'react';
import ReactDOM from 'react-dom';
import App from '../App';
import Index from '../index'

describe('App component', ()=>{
  test ('Should Render the component', ()=>{
    const div = document.createElement('div');
    ReactDOM.render(<App/>, div)
    ReactDOM.unmountComponentAtNode(div);
  });

  test('Index should render without crashing', ()=> {
    expect(JSON.stringify(Object.assign({}, Index, {
      _reactInternalInstance: 'censored'
    }))).toMatchSnapshot();
  });
});