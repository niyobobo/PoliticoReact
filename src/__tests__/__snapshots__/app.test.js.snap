// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component Index should render without crashing 1`] = `"{\\"_reactInternalInstance\\":\\"censored\\"}"`;
